service/src/test/java/uk/gov/hmcts/reform/fpl/validation/validators/IsValidHearingEditValidatorTest.java

```diff
@@ -29,6 +30,32 @@
 
     @Nested
     class EditingHearings {
+        @Test
+        void shouldReturnAnErrorWhenEditingAHearingButNoPastHearingsAreAvailable() {
+            CaseData caseData = CaseData.builder().hearingOption(EDIT_PAST_HEARING).build();
+            List<String> validationErrors = validate(caseData, HearingBookingGroup.class);
+
+            assertThat(validationErrors).contains(ERROR_MESSAGE);
+        }
+
+        @Test
+        void shouldReturnAnErrorWhenEditingAHearingButNoPastHearingsAreAvailable2() {
```

Comment:
maybe rename to something like [...]EditingPastHearingButOnlyFutureHearingsAvailable just to reflect the precondition in the test?